kalite/version.py

```diff
@@ -5,10 +5,15 @@
 # THIS IS USED BY settings.py.  NEVER import settings.py here; hard-codes only!
 # Must be PEP 440 compliant: https://www.python.org/dev/peps/pep-0440/
 # Must also be of the form N.N.N for internal use, where N is a non-negative integer
-MAJOR_VERSION = "0"
-MINOR_VERSION = "14"
-PATCH_VERSION = "0"
-VERSION = "%s.%s.%s" % (MAJOR_VERSION, MINOR_VERSION, PATCH_VERSION)
+
+from django.utils.version import get_version
+
+VERSION_TUPLE = (0, 14, 0, "alpha", 0)
+
+MAJOR_VERSION, MINOR_VERSION, PATCH_VERSION = map(str, VERSION_TUPLE[:3])
+
+VERSION = get_version(VERSION_TUPLE)
```

Comment:
Regarding using letters in the version string, it does what we've been doing with all the alphas and will make all dev work be based on a "123.123.dev123" string... so that part will be well-tested though not so crucial for  releases (unless we do 123.123.post123).